spec/lib/shanty/plugins/rubygem_plugin_spec.rb

```diff
@@ -25,5 +25,18 @@ module Shanty
         subject.build_gem
       end
     end
+
+    describe('#artifacts') do
+      it('lists the project artifacts') do
+        path = File.join(File.dirname(__FILE__), '..', '..', '..', '..')
+        result = subject.artifacts(Project.new(path))
```

Comment:
You should change the:

```ruby
include_context('basics')
```

to

```ruby
include_context('graph')
```

And then modify these two lines to:

```ruby
FileUtils.touch(File.join(project_path, 'foo.gemspec'))
result = subject.artifacts(project_path)
```

Adding the `require 'fileutils'` at the top.

Reasoning here is that we've elsewhere stopped testing using example projects in Shanty itself and have moved to making a temporary folder with more testable mocks. It avoids lots of weird relative path nonsense which makes moving things around a total pain.